Source/AssistantV1/Models/Context.swift

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018.
+ * Copyright IBM Corporation 2019
```

Comment:
This is actually a case where we want to revert all the changes in this file since there are no "substantive" changes.